rise/static/rise.yaml

```diff
@@ -0,0 +1,216 @@
+#
+# NOTE that the defaults defined here are essentially NOT used
+# in the code, but if only for documentation purposes it is important
+# that they reflect the defaults that are hardwired in main.js
+#
+Type: Jupyter Notebook Extension
+Compatibility: jupyter-4.3, jupyter-notebook-5.2
+Name: RISE - Slideshow with reveal.js
+Main: main.js
+Description: "Run notebook as a slideshow thanks to reveal.js"
+Link: configurator.md
+Icon: rise.png
```

Comment:
Thanks for the icon you build. Eventually, we should come up with a nice logo, but it works for now.